test/publish.test.js

```diff
@@ -80,3 +80,20 @@ test('publish (DM to feedId)', t => {
     })
   })
 })
+
+test('list groups', (t) => {
+  const server = Server()
+
+  server.tribes.create(null, (err, data) => {
+    t.error(err)
+    t.deepEqual(server.tribes.list(), [data.groupId])
+    const actualGroup = server.tribes.get(data.groupId)
+    const expectedGroup = {
+      key: data.groupKey,
+      root: data.groupInitMsg.key,
+      scheme: 'envelope-large-symmetric-group'
+    }
+    t.deepEqual(actualGroup, expectedGroup)
+    server.close(t.end)
+  })
+})
```

Comment:
this is a strange file to put this test in (to me).
I'm moving this to `test/api/group/list.test.js`

Reply:
DONE. please pull

Reply:
Thanks. I was overwhelmed at the number of files in `test/` and so I added it to the test that already had a group being created.